VisualForce Page/Assignment 2/AccountSearchPage.cls

```diff
@@ -0,0 +1,52 @@
+<apex:page controller="AccountSearchControllers" >
+    <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <title>Search Account</title>
+        <!-- Import the Design System style sheet -->
+        <apex:slds />
+    </head>
+    <body>
+        <apex:form >
+            <apex:pageblock >
+                <apex:pageBlockButtons location="top">
+                    <apex:commandButton value="Search" action="{!SearchAccountRec}"/>
+                </apex:pageBlockButtons>
+                <apex:pageBlockSection >
+                    <apex:pageBlockSectionItem >
+                        <apex:outputLabel value="BillingCity" />
+                        <apex:inputText value="{!BillingCity}" />
+                    </apex:pageBlockSectionItem>
+                    <apex:pageBlockSectionItem >
+                        <apex:outputLabel value="Number of Accounts To show" />
+                        <apex:inputText value="{!numOfRecords}" />
+                    </apex:pageBlockSectionItem>
+                    <apex:pageBlockSectionItem >
+                        <apex:outputLabel value="BillingState" />
+                        <apex:inputText value="{!BillingState}" />
+                    </apex:pageBlockSectionItem>
+                    <apex:pageBlockSectionItem >
+                        <apex:outputLabel value="BillingCountry" />
+                        <apex:inputText value="{!BillingCountry}" />
+                    </apex:pageBlockSectionItem>
+                </apex:pageBlockSection>
+                <apex:pageBlock rendered="{!If(accounts.size > 0,true,false)}" title="Search Results">

```

Comment:
Can we simply write this condition 
{!IF(accounts.size > 0)}

Reply:
Following error will occur  if i write  condition like this :
Incorrect number of parameters for function 'If()'. Expected 3, received 1